packages/test-helpers/src/asserts/assertEvent.js

```diff
@@ -1,7 +1,17 @@
 const { assert } = require('chai')
 const { isAddress, isBN, toChecksumAddress } = require('web3-utils')
 
-const { getEventAt, getEvents } = require('./events')
+const { getEvents, getEventAt } = require('../')
+
+function normalizeArg(arg) {
+  if (isBN(arg)) {
+    return arg.toString()
+  } else if (isAddress(arg)) {
+    return toChecksumAddress(arg)
+  }
```

Comment:
Let's contemplate the case of comparing contracts with addresses: see https://github.com/aragon/contract-helpers/pull/35#discussion_r453367647

Reply:
Oh interesting, so that's why that was there. I'll add it back!

Reply:
Done in 762277e.